src/backend/web/routes/opml.js

```diff
@@ -1,25 +1,33 @@
 const express = require('express');
 const opml = require('opml-generator');
+const { getFeeds, getFeed } = require('../../utils/storage');
+const { logger } = require('../../utils/logger');
 
 const router = express.Router();
 
 const header = {
   title: 'OPML Feeds',
-  dateCreated: new Date(2020, 2, 9),
-  ownerName: 'Jordan',
+  dateCreated: new Date(),
+  ownerName: 'Telescope',
 };
 
-const outlines = [
-  {
-    text: 'txt',
-    title: 'My Open Source Experience',
-    type: 'rss',
-    xmlUrl: 'https://jrdnlxopensource.blogspot.com/feeds/posts/default/-/open-source',
-    htmlUrl: 'https://jrdnlxopensource.blogspot.com/',
-  },
-];
+router.get('/', async (req, res) => {
+  let feeds;
+  try {
+    feeds = await Promise.all((await getFeeds()).map(feedId => getFeed(feedId)));
+  } catch (error) {
+    logger.error({ error }, 'Failed to get feeds from Redis storage');
+  }
+  const outlines = feeds.map(feed => {
+    return {
+      text: feed.id,
+      title: `${feed.author}'s blog`,
+      type: feed.url.includes('atom') ? 'atom' : 'rss',
```

Comment:
This is probably not the right thing long term, but is fine for now until we figure out a proper way to accomplish it via the parser.